backend/src/routes.js

```diff
@@ -1,5 +1,11 @@
 const express = require('express');
 
+const UserController = require('./controllers/UserController');
+
 const routes = express.Router();
 
-module.exports = routes;
\ No newline at end of file
+routes.post('/createUser', UserController.create);
+routes.delete('/deleteUser/:id', UserController.delete);
+routes.get('/getUsers', UserController.index);
```

Comment:
Suggestion: use the same URL for different routes. Differentiate which controller to call using the request method.

For instance:
`routes.post('/user', UserController.create);`
`routes.delete('/user', UserController.delete);`

Reply:
I have my concerns if this is really a good practice or not. It seems to me that this would result on many methods and calls trough out the system making a call to the same name, maybe not so good for reading purposes.

Reply:
This isn't a good practice. Because of that, i'm ignoring this review.